2022-01-15 15:49:39,627 INFO    Thread-17 :24105 [wandb_setup.py:_flush():71] setting env: {'project': 'Amazon-Reviews', 'entity': 'ewebid', 'root_dir': '/Users/ewelina/Desktop/MLOps_Amazon_review_project', 'run_id': 'rc9odqcr', 'sweep_param_path': '/Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/sweep-ncwmt14p/config-rc9odqcr.yaml', 'sweep_id': 'ncwmt14p'}
2022-01-15 15:49:39,627 INFO    Thread-17 :24105 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-15 15:49:39,627 INFO    Thread-17 :24105 [wandb_init.py:_log_setup():371] Logging user logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154939-rc9odqcr/logs/debug.log
2022-01-15 15:49:39,628 INFO    Thread-17 :24105 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/ewelina/Desktop/MLOps_Amazon_review_project/wandb/run-20220115_154939-rc9odqcr/logs/debug-internal.log
2022-01-15 15:49:39,628 INFO    Thread-17 :24105 [wandb_init.py:init():404] calling init triggers
2022-01-15 15:49:39,628 INFO    Thread-17 :24105 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 250, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
config: {}
2022-01-15 15:49:39,628 INFO    Thread-17 :24105 [wandb_init.py:init():460] starting backend
2022-01-15 15:49:39,629 INFO    Thread-17 :24105 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-15 15:49:39,635 INFO    Thread-17 :24105 [backend.py:ensure_launched():216] starting backend process...
2022-01-15 15:49:39,640 INFO    Thread-17 :24105 [backend.py:ensure_launched():221] started backend process with pid: 24248
2022-01-15 15:49:39,644 INFO    Thread-17 :24105 [wandb_init.py:init():469] backend started and connected
2022-01-15 15:49:39,647 INFO    Thread-17 :24105 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 250, 'drop_out': 0.25, 'epochs': 5, 'lr': 0.01, 'optimizer': 'sgd'}
2022-01-15 15:49:39,651 INFO    Thread-17 :24105 [wandb_init.py:init():533] updated telemetry
2022-01-15 15:49:39,893 INFO    Thread-17 :24105 [wandb_init.py:init():563] communicating current version
2022-01-15 15:49:44,896 INFO    Thread-17 :24105 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-15 15:49:46,035 INFO    Thread-17 :24105 [wandb_init.py:init():606] starting run threads in backend
2022-01-15 15:49:51,061 INFO    Thread-17 :24105 [wandb_run.py:_console_start():1810] atexit reg
2022-01-15 15:49:51,063 INFO    Thread-17 :24105 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-15 15:49:51,064 INFO    Thread-17 :24105 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-15 15:49:51,065 ERROR   Thread-17 :24105 [wandb_run.py:_redirect():1748] Failed to redirect.
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1741, in _redirect
    out_redir.install()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/opt/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/anaconda3/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-01-15 15:49:51,066 INFO    Thread-17 :24105 [wandb_init.py:init():633] run started, returning control to user process
2022-01-15 15:50:00,952 INFO    Thread-17 :24105 [wandb_run.py:finish():1457] finishing run ewebid/Amazon-Reviews/rc9odqcr
2022-01-15 15:50:00,967 INFO    Thread-17 :24105 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-15 15:50:00,976 INFO    Thread-17 :24105 [wandb_run.py:_restore():1752] restore
2022-01-15 15:50:05,455 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
}
pusher_stats {
}

2022-01-15 15:50:05,853 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 772
}

2022-01-15 15:50:06,127 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18071
}

2022-01-15 15:50:06,231 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18071
}

2022-01-15 15:50:06,335 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18071
}

2022-01-15 15:50:06,451 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18071
}

2022-01-15 15:50:06,557 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 772
  total_bytes: 18071
}

2022-01-15 15:50:06,663 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:06,770 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:06,876 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:06,984 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:07,087 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:07,191 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:07,929 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}

2022-01-15 15:50:08,633 INFO    Thread-17 :24105 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 18071
  total_bytes: 18071
}
local_info {
}

2022-01-15 15:50:09,871 INFO    Thread-17 :24105 [wandb_run.py:_append_files():2180] logging synced files
